### PR TITLE
Load files into their respective classes

### PR DESCRIPTION
**Pull request by hs0084**
Initial attempt in loading the CSV files into their correct objects.

### CODE REVIEW

File: header/department.h [hunk 1]
```diff
@@ -9,7 +9,7 @@ using namespace std;
 class Department {
 public:
 	vector<Course> courseList;
-	vector<Instructor> instructiorList;
+    vector<Instructor> instructorList;
```
Comment (Campbell-Lindsay) [inline header/department.h:12]: Weird Spacing?

Comment (hs0084) [reply header/department.h:12]: Looks normal in both VS and QT, only GitHub does this.

**Review by Campbell-Lindsay — COMMENTED** [review]
Looks good, from what I can tell!

**Comment by Actual-Heathen** [comment]
code should compile now

File: header/instructor.h [hunk 2]
```diff
@@ -7,6 +7,7 @@ class Instructor {
 private:
 	string firstName;
 	string lastName;
+    int maxCourses;
 	int priority;
 	string preferredClass;
 	bool conflict = false;
```
Comment (jmp0060) [inline header/instructor.h:13]: conflict should exist only within the Course object, not within instructor

File: header/instructor.h
Comment (jmp0060) [inline header/instructor.h:1]: Instructor needs its own DayTime availability object

File: source/resourceManager.cpp [hunk 3]
```diff
@@ -7,6 +9,10 @@ void resourceManager(bool populated, int departmentCounter, string fileStoragePa
     string course[10];
```
Comment (jmp0060) [inline source/resourceManager.cpp:9]: Whatever this number is - I assume it's an arbitrary limit - it should be saved as a variable, not hard coded. Use const int limit = 10;

File: source/resourceManager.cpp
Comment (jmp0060) [inline source/resourceManager.cpp:1]: This looks like it is only creating one department object. I'm not seeing where the list of departments is populated as it should be.

**Review by jmp0060 — CHANGES_REQUESTED** [review]

File: header/dayTime.h [hunk 4]
```diff
@@ -4,9 +4,9 @@ using namespace std;
 
 class DayTime {
 private:
-	bool times[2][7] = { {false, false, false, false, false, false, false}, {false, false, false, false, false, false, false} };
+    int times[2][7] = { {0, 0, 0, 0, 0, 0, 0}, {0, 0, 0, 0, 0, 0, 0} };
```
[the remark this answers is not in the document]
Comment (hs0084) [reply header/dayTime.h:7]: Even if I create const int days = 2, int times[days] still gives the error "a nonstatic member reference must be relative to a specific object".

**Review by Actual-Heathen — APPROVED** [review]
sure

**Review by Actual-Heathen — APPROVED** [review]

**Pull request merged**